test/4.public.UrlChecker.js

```diff
@@ -139,9 +139,83 @@ describe("PUBLIC -- UrlChecker", function()
 			}).enqueue( conn.absoluteUrl );
 		});
 	});
-	
-	
-	
+
+
+
+	describe("caching", function()
+	{
+		var options;
+		beforeEach(function()
+		{
+			options = utils.options();
```

Comment:
Oh, I was just being snotty about a tiny style preference. I lean towards trying to reduce duplication over pretty much everything. I have no problem with this :)

Sorry for being snooty.